QuantConnect.DataBento/DataBentoDataProvider.cs

```diff
@@ -113,64 +86,35 @@ public DataBentoProvider(string apiKey)
+                        return false;
+                    }
 
-                        if (_client?.IsConnected == true)
-                        {
-                            Log.Trace($"DataBentoProvider.SubscribeImpl(): Client is connected, attempting async subscribe for {symbol}");
-                            Task.Run(async () =>
-                            {
-                                // If the requested resolution is higher than tick, we subscribe to ticks and let the aggregator handle it.
-                                var resolutionToSubscribe = config.Resolution > Resolution.Tick ? Resolution.Tick : config.Resolution;
-                                var success = _client.Subscribe(config.Symbol, resolutionToSubscribe, config.TickType);
-                                if (success)
-                                {
-                                    Log.Trace($"DataBentoProvider.SubscribeImpl(): Successfully subscribed to {config.Symbol} at {resolutionToSubscribe} resolution");
-                                    
-                                    // Start session after first successful subscription
-                                    lock (_sessionLock)
-                                    {
-                                        if (!_sessionStarted)
-                                        {
-                                            Log.Trace("DataBentoProvider.SubscribeImpl(): Starting DataBento session to receive data");
-                                            _sessionStarted = _client.StartSession();
-                                            if (_sessionStarted)
-                                            {
-                                                Log.Trace("DataBentoProvider.SubscribeImpl(): Session started successfully - data should begin flowing");
-                                            }
-                                            else
-                                            {
-                                                Log.Error("DataBentoProvider.SubscribeImpl(): Failed to start session");
-                                            }
-                                        }
-                                    }
-                                }
-                                else
-                                {
-                                    Log.Error($"DataBentoProvider.SubscribeImpl(): Failed to subscribe to live data for {config.Symbol}");
-                                }
-                            });
-                        }
-                        else
-                        {
-                            Log.Trace($"DataBentoProvider.SubscribeImpl(): Client not connected, skipping subscription for {symbol}");
-                        }
+                    var resolution = config.Resolution > Resolution.Tick ? Resolution.Tick : config.Resolution;
+                    if (!_client.Subscribe(config.Symbol, resolution, config.TickType))
```

Comment:
We shouldn't use `resolution` and `config.TickType` when subscribing to Live data.

https://databento.com/docs/schemas-and-data-formats/whats-a-schema#supported-schemas-and-their-fields?historical=python&live=python&reference=python